lib/iris/analysis/__init__.py

```diff
@@ -1296,6 +1296,10 @@ def _calc_percentile(data, percent, fast_percentile_method=False, **kwargs):
     Calculate percentiles along the trailing axis of a 1D or 2D array.
 
     """
+    # Get hold of method keyword to pass it to numpy.percentile, or just remove it
+    # from kwargs so it's ignored for scipy mquantiles.  Note that kwargs always
+    # contains alphap and betap so we can't pass it to numpy.percentile.
+    method = kwargs.pop("method", "linear")
```

Comment:
Thinking about it, this couples us to `np.percentile`'s default - can we instead only set `method` in the `np.percentile` call if it's user specified and if not just not specify it? Means our wrapper is thinner, which seems like a good direction to go in if it's not too faffy